### PR TITLE
App: Directory - Added logic to create app folders

### PR DESCRIPTION
**Pull request by techmsi**
https://trello.com/c/RLQFln2t/6-generate-clay-folders

### App Folders
* `sites`
* `components`
* `behaviors`
* `validators`

### CODE REVIEW

**Comment by yoshokatana** [comment]
:+1:

**Comment by yoshokatana** [comment]
Is this ready to merge?

**Comment by techmsi** [comment]
@yoshokatana Not yet. One little piece left.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ This will add the following files:
...
+#### Example
+
+```
+- Sites Folder (exists)
+
+- Components Folder (exists)
+
+- Behaviors Folder (exists)
+
+❯ ◉ Validators Folder
```
Comment (yoshokatana) [inline README.md]: This doesn't need to be part of the menu. Just create the folders when it runs.

**Comment by yoshokatana** [comment]
No tests?

File: README.md (hunk 1 above)
Comment (techmsi) [inline README.md]: Shouldn't we give users the choice of whether or not to create these folder?

**Comment by techmsi** [comment]
@yoshokatana Should I add a tests folder? That's not listed on the ticket.

**Comment by yoshokatana** [comment]
No, no. I mean tests for the generator itself. e.g. https://github.com/nymag/generator-clay/blob/master/generators/component/test.js

File: README.md (hunk 1 above)
Comment (yoshokatana) [inline README.md]: There's no need. We're already being opinionated about what folders people should have.

Comment (techmsi) [inline README.md]: Ok.

**Comment by techmsi** [comment]
Ok.

**Comment by techmsi** [comment]
Tests added.

**Comment by yoshokatana** [comment]
:+1:

**Pull request merged**